src/styles/block/header.scss

```diff
@@ -0,0 +1,71 @@
+.header {
+  flex-basis: 1024px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: $back-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+
+  &__img {
+    width: 100%;
+    height: 100%;
+  }
+}
+
+.nav {
+  display: flex;
+
+  &__list {
+    display: flex;
+    justify-self: center;
+    list-style: none;
+    padding: 0;
+  }
+
+  &__item {
+    margin-left: 20px;
+  }
+
+  &__item:first-child {
```

Comment:
Use the ':first-child' pseudo-class inside '&__item' with the '&' symbol, like '&:first-child'.